test/pg_prove.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+# Wrapper around perl prove utility to control running of TAP tests
+#
+# The following control variable is supported:
+#
+# PROVE_TESTS  only run TAP tests from this list
+# e.g make provecheck PROVE_TESTS="t/foo.pl t/bar.pl"
+#
+# Note that you can also use regular expressions to run multiple
+# taps tests matching the pattern:
+#
+# e.g make provecheck PROVE_TESTS="t/*chunk*"
+#
+
+PROVE_TESTS=${PROVE_TESTS:-}
+PROVE=${PROVE:-prove}
+
+echo "PROVE_TESTS ${PROVE_TESTS}"
```

Comment:
Looks like debugging. Should we always show this? Most of the times it is empty I guess?

Reply:
Yeah, more like debugging. Can remove it.